Makefile

```diff
@@ -38,7 +38,7 @@ check_license:
 
 test:
 	@echo ">> running tests"
-	@$(GO) test -short $(pkgs)
+	@$(GO) test -race -short $(pkgs)
```

Comment:
This increases the time to run the tests and their memory usage significantly. Given that we often run those tests resource-constrained and already pass in the `-short` flag to not run the more expensive tests, we should not make _all_ tests 10x more expensive (even those that cannot detect races by definition because they are non-concurrent).

Perhaps keep the test target as it is and add a new `test_race` target?